src/emqx_client.erl

```diff
@@ -787,7 +787,7 @@ connected(cast, ?SUBACK_PACKET(PacketId, Properties, ReasonCodes),
 connected(cast, ?UNSUBACK_PACKET(PacketId, Properties, ReasonCodes),
           State = #state{subscriptions = Subscriptions}) ->
     case take_call({unsubscribe, PacketId}, State) of
-        {value, #call{from = From, req = {_, Topics}}, NewState} ->
+        {value, #call{from = From, req = {_, _, Topics}}, NewState} ->
```

Comment:
this would be easier to read/troubleshoot if a macro is used here.
e.g. `?UNSUB_REQ(_, _, Topics)`